content/docs/hoon/reference/rune/tis.md

```diff
@@ -4,127 +4,286 @@ weight = 5
 template = "doc.html"
 +++
 
-These runes modify the subject.  (Or more precisely, they
+These runes modify the subject. (Or more precisely, they
 evaluate at least one of their subexpressions with a modified subject.)
 
 ## Overview
 
-Hoon doesn't have variables in the ordinary sense.  If you want to bind a name
+Hoon doesn't have variables in the ordinary sense. If you want to bind a name
 to a value, e.g., `a` to `12`, you do so by pinning `12` to the subject and
-associating the name with it.  This sort of operation is done with the `=`
+associating the name with it. This sort of operation is done with the `=`
 family of runes.
 
-Let's say you have some old subject `p`.  To 'pin' a value to the head means to
-modify the subject by repacing it with a cell of `[new-value p]`.  The head of
-the cell is the new value.  So to pin `12` with the face `a` the new subject
+Let's say you have some old subject `p`. To 'pin' a value to the head means to
+modify the subject by repacing it with a cell of `[new-value p]`. The head of
+the cell is the new value. So to pin `12` with the face `a` the new subject
 would be: `[a=12 p]`.
 
 Of course there are many variations on ways to modify the subject, useful for
-different situations.  Hence the whole family of `=` runes.
+different situations. Hence the whole family of `=` runes.
+
+---
+
+## `=>` "tisgar"
+
+Compose two expressions.
+
+#### Syntax
+
+Two arguments, fixed.
+
+<table>
+<tr><th>Form</th><th>Syntax</th></tr>
+<tr>
+<td>Tall</td>
+<td>
+<pre>
+=>  p
+q
+</pre>
+</td>
+</tr>
+<tr>
+<td>Wide</td>
+<td>
+<pre>
+=>(p q)
+</pre>
+</td>
+</tr>
+<tr>
+<td>Irregular</td>
+<td>None.</td>
+</tr>
+</table>
+
+#### AST
 
-## Runes
-
-### `=>` "tisgar"
-
-`[%tsgr p=hoon q=hoon]`: compose two expressions.
+```hoon
+[%tsgr p=hoon q=hoon]
+```
 
-##### Produces
+#### Produces
 
 the product of `q`, with the product of `p` taken as the subject.
 
-##### Syntax
-
-Regular: **2-fixed**.
-
-##### Examples
+#### Examples
 
 ```
 > =>([a=1 b=2 c=3] b)
 2
 
+> =>  9  .
+9
+
 > =>((add 2 4) [. .])
 [6 6]
-```
 
-### `=|` "tisbar"
+> =>
+  |%
+  ++  foo
+    |=  [a=@ b=@]
+    [b a]
+  --
+  (foo 42 27)
+[27 42]
+```
 
+---
+
+## `=|` "tisbar"
+
+Combine a default type value ("bunt value") with the subject.
```

Comment:
This phrasing seems confusing to new learners... let's pattern it after =/ and =+ which “combine a named noun with the subject”:

“Combine a named noun with the subject; assign it the bunt value” or similar